src/pages/index.jsx

```diff
@@ -1,12 +1,18 @@
-import React from 'react';
-import Header from '../components/Header';
-import ContentContainer from '../components/ContentContainer';
-import TodoForm from '../components/TodoForm';
-import TodoList from '../components/TodoList';
-
-function IndexPage() {
+import React, { useEffect } from "react";
+import Header from "../components/Header";
+import ContentContainer from "../components/ContentContainer";
+import TodoForm from "../components/TodoForm";
+import TodoList from "../components/TodoList";
+import styles from "./styles.module.sass";
+import { connect } from "react-redux";
+import { todosAction } from "../redux/actions";
+function IndexPage({ fetchTodos }) {
+  useEffect(() => {
+    fetchTodos();
+    // eslint-disable-next-line
```

Comment:
This is not how you fix eslint errors. You should have added `fetchTodos` as a dependency.